app/helpers.php

```diff
@@ -55,12 +55,12 @@ function correspondence($message = null, $field = null)
  * @param  string  $defaut
  * @return string
  */
-function format_date_form_field($model, $field, $defaut = 'MM/DD/YYYY')
+function format_date_form_field($model, $field, $default = 'MM/DD/YYYY')
 {
     $oldDate = old($field);
 
     if ($oldDate) {
-        return $oldDate;
+        return (new Carbon($oldDate))->format('Y-m-d');
```

Comment:
so this isn't really needed. The "old" value would be whatever was in the input as the value prior, so there shouldn't be a need to re-carbonize and reformat it.